### ISSUE TITLE
Allow bounce property greater than 1

### ISSUE THREAD

**Issue by Oranjoose** [godotengine/godot#24414]
Currently the `bounce` property of RigidBody nodes must be between 0 and 1, where all energy is lost for a bounce of 0 and no energy is lost at a bounce value of 1. It would be nice if the `bounce` property could have a value greater than 1, where for example a bounce value of 2 would double the energy returned to the RigidBody, thus having somewhat of a launching effect (think Pinball bumpers). I have seen this in other game engines.

**Comment by akien-mga** [comment]
Not surely how useful that would be as your ball's speed would quickly diverge towards +∞. But if it's supported by Bullet, we can always expose it while keeping the inspector field slider from 0 to 1 (using `allow_greater` in the hint to allow manual higher input).

CC @AndreaCatania

**Comment by kakoeimon** [comment]
@Oranjoose  Isn't this possible and more logical, to use the body_exited signal?
I mean that even in real life the bumpers are responsible for the extra bounce of the ball.

**Comment by AndreaCatania** [comment]
@kakoeimon is correct, increase bouncing limit is not a good idea

**Comment by Oranjoose** [comment]
> @kakoeimon: Isn't this possible and more logical, to use the body_exited signal? I mean that even in real life the bumpers are responsible for the extra bounce of the ball.

Yes, and as such you can set the `bounce` property on the StaticBody bumpers, which in turn affect the ball. And by body_exited, do you mean to multiply the the linear_velocity by 1.2 in a body_exited callback, instead of having a bounce of 1.2?

The point is, consider the inverse. If there wasn't already a hard upper limit, then there wouldn't be a particularly strong reason to add one. If someone wants to set a bounce property to more than 1, let them. As an anecdote, in the game development summer camp I run, it seems to be somewhat of a rite of passage that as soon as we go into physics bodies, the teenagers immediately--without being told--increase bounce past 100% and have a blast doing so (different game engine than Godot).

**Comment by 90PlusWorks** [comment]
I may come back to this

**Comment by Calinou** [comment]
PhysicsMaterial already allows manually entering a Bounce value greater than 1.0: https://github.com/Calinou/godot/blob/df936cc9336bb9c572d5e6c1f7cd31993ded5855/scene/resources/physics_material.cpp#L48

However, PhysicsBody3D does not: https://github.com/Calinou/godot/blob/df936cc9336bb9c572d5e6c1f7cd31993ded5855/scene/3d/physics_body_3d.cpp#L2994

**Comment by 90PlusWorks** [comment]
You could try to see if my version works for you...I only worked in 2d though. If similar changes are done in the 3d code?
https://github.com/90PlusWorks/godot-air forked from Godot in 2022.
